README.md

```diff
@@ -1 +1,74 @@
-# codepen-blog-post
+We all know that our beloved Javascript language is somewhat insecure, so errors at runtime (hello 'undefined' is not an object) occur very frequently when systems grow in complexity.
```

Comment:
Acá pondría el error "Cannot get property of undefined" o no?

Reply:
Hablaría más de fail-safe o algo distinto a seguridad, que se entiende como vulnerabilidades. Quizás seguridad de ejecución en runtime?

Reply:
`We all know that our beloved Javascript is somewhat insecure (?). More often than not we encounter unintended errors at runtime (hello...).`

Reply:
eso suena perfect!